website/docs/filter-group/filter-cp-cd-cpc/examples/basic-example.tsx

```diff
@@ -7,30 +7,38 @@ import NeighborLocation from 'intergalactic/neighbor-location';
 import InputNumber from 'intergalactic/input-number';
 import { Text } from 'intergalactic/typography';
 
-const InputRange = ({ value: valueState, changeValue, ariaLabelledby, ...other }) => {
+const InputRange = ({ value: valueState, changeValue, ...other }) => {
   const minRange = 1;
   const maxRange = 8;
-  let revertValues = false;
+
+  const fromRef = useRef(null);
+  const toRef = useRef(null);
 
   const handleChange = (key) => (value) => {
-    if (revertValues) {
-      revertValues = false;
-    } else {
-      valueState[key] = value;
-      changeValue({ ...valueState });
-    }
+    valueState[key] = value ? Number(value) : '';
+    changeValue({ ...valueState });
   };
 
   const handleBlur = () => {
-    const { from, to } = valueState;
-    if (from > to) {
-      revertValues = true;
-      changeValue({
-        from: Math.max(to, minRange),
-        to: Math.min(from, maxRange),
-      });
-    }
+    setTimeout(() => {
+      if (document.activeElement !== fromRef.current && document.activeElement !== toRef.current) {
+        const { from, to } = valueState;
+        if (from > to && to) {
+          changeValue({
+            from: Math.max(to, minRange),
+            to: Math.min(from, maxRange),
+          });
+        }
+        if ((!to || !from) && (to || from)) {
```

Comment:
Not sure about this, because `0` is false. And about condition on 26 line too (`&& to` could be invalid if `to=0`)

Reply:
yeah you're right, minRange can be zero, fixed it

Reply:
I fixed it a little in another places. Everything is good

Reply:
oops, thanks!